src/RefractAPI.cc

```diff
@@ -332,63 +380,77 @@ namespace drafter {
         return element;
     }
 
-    refract::IElement* ActionToRefract(const snowcrash::Action& action)
+    refract::IElement* ActionToRefract(const SectionInfo<snowcrash::Action>& action)
     {
         refract::ArrayElement* element = new refract::ArrayElement;
         RefractElements content;
 
         element->element(SerializeKey::Transition);
-        element->meta[SerializeKey::Title] = refract::IElement::Create(action.name);
+        element->meta[SerializeKey::Title] = PrimitiveToRefract(MAKE_SECTION_INFO(action, name));
 
-        if (!action.relation.str.empty()) {
-            element->attributes[SerializeKey::Relation] = refract::IElement::Create(action.relation.str);
+        if (!action.section.relation.str.empty()) {
+            // FIXME: add SourceMap
+            element->attributes[SerializeKey::Relation] = refract::IElement::Create(action.section.relation.str);
         }
 
-        if (!action.uriTemplate.empty()) {
-            element->attributes[SerializeKey::Href] = refract::IElement::Create(action.uriTemplate);
+        if (!action.section.uriTemplate.empty()) {
+            // FIXME: This doues not chnge rendering, do we have exammple with uriTemplate?
```

Comment:
There is no difference between output before and after new code is applied.
Will clear comment.